### PR TITLE
Improve compatibility with Devpi 5.x

### PR DESCRIPTION
**Pull request by theMarix**
This will allow to fix the currently broken master of the Devpi acceptance tests.

### CODE REVIEW

**Comment by theMarix** [comment]
Finally managed to fix all those Python version compatibility issues. So this should be good to review now.

File: devpi_plumber/server.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def TestServer(users={}, indices={}, config={}, fail_on_output=['Traceback']):
 
 def import_state(serverdir, importdir):
     devpi_server_command(serverdir=serverdir, init=None)
-    devpi_server_command(serverdir=serverdir, **{'import': importdir})
+    devpi_server_command(serverdir=serverdir, **{'import': importdir, 'no-events': None})
```
Comment (StephanErb) [inline devpi_plumber/server.py]: What part of the Devpi 5.x release is this related to? 🤔 

https://github.com/devpi/devpi/blob/master/server/CHANGELOG

Comment (StephanErb) [reply devpi_plumber/server.py]: Ah. Finally saw the commit message.
